src/Datadog.Trace/Tracer.cs

```diff
@@ -315,6 +317,7 @@ public Span StartSpan(string operationName, ISpanContext parent = null, string s
             }
             else
             {
+                isRootSpan = true;
```

Comment:
This code path (`isRootSpan = true`) will not be hit for the local root of a distributed trace, only for the root of non-distributed traces. Was that intentional?

Reply:
No, it was not, would it be better to use this, above? `var isRootSpan = parent == null;`